Assets/UXF/Scripts/UI/ParticipantListSelection.cs

```diff
@@ -247,18 +247,16 @@ public string Finish()
 
             // update row in table
             foreach (var keyValuePair in completedForm)
-                row[keyValuePair.Key] = keyValuePair.Value;     
+                UpdateDatapoint(row, keyValuePair.Key, keyValuePair.Value);    
```

Comment:
This will cause a warning to be raised, not an exception, no? Because of:

https://github.com/immersivecognition/unity-experiment-framework/blob/b8dcd5b510ffaac3afcae94370159d9710e600c1/Assets/UXF/Scripts/UI/ParticipantListSelection.cs#L264-L269

I think a warning is OK because it is good that a new column gets added if one does not already exist.

Reply:
Yes, a warning (and not an exception) is logged at the end. I saw the ArgumentException and just named it accordingly. My naming is a bit misleading.

Reply:
On going through this part again, can it even lead to exceptions? As the columns are taken from the completedForm, will it encounter new columns? If yes, then there is sense in accounting for the exception, I think? Then I create another method with that check.

Reply:
Otherwise we can close the pull request.

Reply:
I think you're right, the columns should always exist in this case